server/views/pages/new-base-client-details.njk

```diff
@@ -24,7 +24,7 @@
 {% block content %}
 
   <div class="govuk-grid-row">
-    <form class="govuk-grid-column-two-thirds" action="/base-clients/new" method="POST">
+    <form class="govuk-grid-column-two-thirds" action="/base-clients/new?grant={{ grant }}" method="POST">
```

Comment:
It would be worth checking with Alan if this will still work there is a spring update that is waiting that doesn't allow path variables